MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -1899,6 +1899,9 @@ public string GetVideoQualityParam(EncodingJobInfo state, string videoEncoder, E
                         param += " -prio_speed 1";
                         break;
                 }
+
+                param += " -qmin -1";
+                param += " -qmax -1";
```

Comment:
How about moving these to `GetVideoBitrateParam()` ? They are highly correlated.
https://github.com/jellyfin/jellyfin/blob/a49f552d6ec81c68a5d11f4ce585f824bf5f3150/MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs#L1400-L1406

Reply:
Other encoders are setting the q value here so I chose to do the same as others. If move this to `GetVideoBitrateParam()` is really much better I could do that.

Reply:
https://github.com/jellyfin/jellyfin/blob/a49f552d6ec81c68a5d11f4ce585f824bf5f3150/MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs#L1991

`GetVideoBitrateParam()` is called in `GetVideoQualityParam()`.

In fact, we don't expect them to be called separately, resulting in video quality being partially controlled. It makes more sense to change `GetVideoBitrateParam()` to private.

Reply:
Done.